src/lib/components/Checkbox/Checkbox.tsx

```diff
@@ -16,10 +16,13 @@ export interface CheckboxProps extends Omit<ComponentProps<'input'>, 'type' | 'r
   theme?: DeepPartial<FlowbiteCheckboxTheme>;
 }
 
-export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
+const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
```

Comment:
Is there a functional difference here or just aesthetic? Like, does the displayName still work even if the export is on this line? Currently the rest of the library is written this way so I don't see a reason to change these components unless they aren't actually working

Reply:
There is no functional difference. I'll change it back and push in a bit.

Reply:
No worries, I personally think yours makes more sense as I prefer to read code top to bottom, but it's less work to keep the current convention